File: ios/CalSmartconfig.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(CalSmartconfig, NSObject)

RCT_EXTERN_METHOD(provision:(* NSString)ssid bssid:(* NSString)bssid password:(* NSString)password
                 withResolver:(RCTPromiseResolveBlock)resolve
                 withRejecter:(RCTPromiseRejectBlock)reject)

@end

File: ios/CalSmartconfig.swift
@objc(CalSmartconfig)
class CalSmartconfig: NSObject {

    @objc(provision:location:withResolver:withRejecter:)
    func provision(ssid: String, bssid: String, password: String, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock ) -> Void{

    }
}
